docs/faq.md

```diff
@@ -1,5 +1,137 @@
+
+- `worker-threads` (named `workers` for [external-resizer](https://github.com/kubernetes-csi/external-resizer))
+- `kube-api-burst`
+- `kube-api-qps`
+- `timeout`
+
+If your workloads need a higher throughput of volume operations, or you have higher AWS limits for any of the EC2 APIs in the `EC2 API Calls Made By Driver` column, you may increase consider increasing the associated sidecar's `worker-threads` value. 
+
+| Sidecar | Configuration Name | Description | EC2 API Calls Made By Driver |
+| ---- | ---- | ---- | ---- |
+| [external-provisioner](https://github.com/kubernetes-csi/external-provisioner) | provisioner | Watches PersistentVolumeClaim objects and triggers CreateVolume/DeleteVolume | EC2 CreateVolume/DeleteVolume |
+| [external-attacher](https://github.com/kubernetes-csi/external-attacher) | attacher | Watches VolumeAttachment objects and triggers ControllerPublish/Unpublish | EC2 AttachVolume/DetachVolume |
+| [external-resizer](https://github.com/kubernetes-csi/external-resizer) | resizer | Watches PersistentVolumeClaims objects and triggers controller side expansion operation | EC2 ModifyVolume, EC2 DescribeVolumesModifications |
+| [external-snapshotter](https://github.com/kubernetes-csi/external-snapshotter) | snapshotter | Watches Snapshot CRD objects and triggers CreateSnapshot/DeleteSnapshot | EC2 CreateSnapshot/DeleteSnapshot, EC2 DescribeSnapshots |
+
+#### Sidecar Fine-tuning Examples
+
+**Note: The external-resizer uses the `--workers` parameter instead of `--worker-threads`
+##### Helm
+
+Make the following diff in the helm chart's default [values.yaml](https://github.com/kubernetes-sigs/aws-ebs-csi-driver/blob/master/charts/aws-ebs-csi-driver/values.yaml) file
+
+```diff
+sidecars:
+  provisioner:
+    ...
+    # Additional parameters provided by external-provisioner.
+-   additionalArgs: []
++   additionalArgs:
++   - "--worker-threads=101"
++   - "--kube-api-burst=200"
++   - "--kube-api-qps=40.0"
++   - "--timeout=61s"
+...
+  resizer:
+-   additionalArgs: []
++   additionalArgs:
++   - "--workers=101"
+```
+
+##### EKS Managed Add-on
+
+Create a file named `example-addon-config.yaml` with the following yaml:
+
+```yaml
+sidecars:
+  attacher:
+    additionalArgs:
+    - "--worker-threads=101"
+    - "--kube-api-burst=200"
+    - "--kube-api-qps=40.0"
+    - "--timeout=61s"
+  resizer:
+    additionalArgs:
+    - "--workers=101"
+```
+
+Pass in the add-on configuration-values file when creating your add-on: 
+
+```
+ADDON-CONFIG-FILEPATH="./example-addon-config.yaml"
+
+aws eks create-addon \
+  --cluster-name "example-cluster" \
+  --addon-name "aws-ebs-csi-driver" \
+  --service-account-role-arn "arn:aws:iam::123456789012:role/EBSCSIDriverRole" \
+  --configuration-values "file://$ADDON-CONFIG-FILEPATH"
+```
+
+You can confirm that these arguments were set by describing am `ebs-csi-controller` pod and observing the following args under the relevant sidecar container:
```

Comment:
np: `am` -> `a`

Reply:
additional np: "You can confirm ..." -> "Confirm that..."